scripts/provision.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
+
+set -e # fail on error
+
+SCRIPT_DIR=$(dirname $0)
+
+# Include functions to setup GCE BOSH networking
+. $SCRIPT_DIR/gce-assign-fixed-ip.sh
+
+# Read the platform configuration
+TARGET_PLATFORM=$1
+case $TARGET_PLATFORM in
+  aws)
+    STEMCELL=light-bosh-stemcell-3069-aws-xen-hvm-ubuntu-trusty-go_agent.tgz
+    ;;
+  gce)
+    STEMCELL=light-bosh-stemcell-2968-google-kvm-ubuntu-trusty-go_agent.tgz
+    STEMCELL_URL=http://storage.googleapis.com/bosh-stemcells/$STEMCELL
+    ;;
+  *)
+    echo "Must specify the target platform: gce|aws"
+    exit 1
+    ;;
+esac
+
+# Convert the yaml outputs file to a list of environment variables:
+#
+#   terraform_output_bastion_ip=104.155.62.123
+#   terraform_output_bosh_ip=104.155.37.66
+#
+# etc...
+eval $(
```

Comment:
Yes you are right, it is too hacky, and it can break if there is a multiline variable, for instance. I will change that.